MANIFEST.in

```diff
@@ -1,10 +1,12 @@
-include edl-v10 epl-v10
+include edl-v10
+include epl-v20
 include README.rst
 include CONTRIBUTING.md
-include setup.py
 include notice.html
 include LICENSE.txt
 include about.html
 
 recursive-include src *.py
 recursive-include examples *.py
+global-exclude __pycache__
```

Comment:
⛔ below is because GitHub expects all text files to end with one and only one blank line.

Reply:
Hi @cclauss Apologies if I'm missing something here, but I'm not seeing any specific errors or warnings in my build workflow with regard to missing blank lines in MANIFEST.in. Also, if you add blank lines to any json file, black will promptly remove them again! What exactly is the issue? Thanks.

Reply:
Click the link in my note above and then look just below line 12.